src/Directory.Build.props

```diff
@@ -1,18 +1,18 @@
 <Project>
   <PropertyGroup>
-    <Version>0.16.0</Version>
+    <Version>0.17.0</Version>
```

Comment:
I think 0.17.0 is already released (https://www.nuget.org/packages/AngleSharp.Diffing) - we could make this 0.17.1.

Reply:
Yeah, actually 1.18 is already released I guess. Not sure how that happened without this being bumped to 1.8.x?

Reply:
```
0.18.0-alpha-40 | 692 | 3/17/2022 |  
-- | -- | -- | --
0.18.0-alpha-38 | 691 | 3/15/2022 |  
0.17.0 | 1,223,976 | 9/8/2021
```

It's just a preview of 0.18.0 No need to follow that through. Just make it 0.17.1 and you are good.

Reply:
will do